src/playlist-loader.js

```diff
@@ -304,7 +313,7 @@ export default class PlaylistLoader extends EventTarget {
     // merge this playlist into the master
     const update = updateMaster(this.master, playlist);
 
-    this.targetDuration = playlist.targetDuration;
+    this.targetDuration = playlist.partTargetDuration || playlist.targetDuration;
```

Comment:
We use a playlists targetDuration in a few places outside of these files. In the case where we have partTargetDuration that is always what we will use as out "true" targetDuration, I understand that this is ambiguous though and I think we might want to rename `this.targetDuration` to something else, but that might be a breaking change. Should we deprecate this name?